docker/regtest/docker-compose.yml

```diff
@@ -99,7 +99,7 @@ services:
   irc:
     container_name: jm_regtest_irc
     restart: unless-stopped
-    image: ghcr.io/ergochat/ergo:stable
+    image: ghcr.io/ergochat/ergo:v2.10.0
```

Comment:
What's the benefit of switching to an explicit version here, instead of `stable` or `latest`?

Reply:
I though it might prevent unintended side-effects of newer versions in the future. Kind of like "version pinning". Dont want to run in the situation where the regtest setup fails for users just to figure out the irc mechansim is not working - suppose it is rather hard to find if one did not regenerate the setup for a while. It not an important change and I can revert it, if you think we should stay on the `stable` image.